client/my-sites/checkout/composite-checkout/components/prepurchase-notices/index.tsx

```diff
@@ -100,23 +130,22 @@ const PrePurchaseNotices = () => {
 		return null;
 	}
 
-	// We're attempting to buy Jetpack Backup individually,
-	// but this site already has a plan that includes it.
-	// ignore the error on free plans as they do not include any paid features
 	if ( currentSitePlan && cartProductThatOverlapsSitePlan ) {
 		return (
-			<SitePlanIncludesCartProductNotice
-				plan={ currentSitePlan }
-				product={ cartProductThatOverlapsSitePlan }
-				selectedSite={ selectedSite }
-			/>
+			<>
+				<SitePlanIncludesCartProductNotice
+					plan={ currentSitePlan }
+					product={ cartProductThatOverlapsSitePlan }
+					selectedSite={ selectedSite }
+				/>
+			</>
```

Comment:
I don't think we need this Fragment here, I assume it's probably leftover from some other thing you were trying 😄
```suggestion
			<SitePlanIncludesCartProductNotice
				plan={ currentSitePlan }
				product={ cartProductThatOverlapsSitePlan }
				selectedSite={ selectedSite }
			/>
```

Reply:
Yes, my mistake. Good catch, thanks Dylan!  Committed. 👍